tests/integration/data/valid/on-and-to.on-amd64-to-riscv64.yaml

```diff
@@ -0,0 +1,10 @@
+plugin: nil
+source-branch: branch1
+build-packages:
+  - package1
+  - package2
+  - package3
+  - package4
+  - package9
+organize:
+  file1: file1
```

Comment:
OK this seems like an actual bug to me. Surely on amd64 to riscv64 should say:

```yaml
organize:
  file1: file1
  file2: file2
  file3: file3
  file4: file4

Reply:
Bugs coming as a follow-up.